KokoDajMu/KokoDajMu_MAIN/Classes/WriteInFile.cs

```diff
@@ -0,0 +1,130 @@
+
+            using (convertListenerListInTextFile)
+            {
+                foreach (Listener listener in this.listenersList)
+                {
+                    convertListenerListInTextFile.WriteLine("<listener><{0}><{1}>[{2}](genres: [{3}])(likedSongs: [{4}])(playlists: [{5}])</listener>",
+                        listener.UserName, listener.FullName, listener.DateOfBirth, String.Join(", ", listener.genres), String.Join(", ", listener.FavoriteSongs), String.Join(", ", listener.playlists.Select(playlistNames => playlistNames.Name)));
+                }
+            }
+        }
+
+        public void SaveArtistListInTextFile()
+        {
+            TextWriter convertArtistListInTextFile = new StreamWriter(@"\\files\Public\Transfer\Kaloian Karaivanov\Internship\internship_tasks\KokoDajMu\KokoDajMu_MAIN\TextDocuments\ArtistList.txt");
+
+            using (convertArtistListInTextFile)
+            {
+                foreach (Artist artist in this.artistsList)
+                {
+                    convertArtistListInTextFile.WriteLine("<artist><{0}><{1}>[{2}](genres:[{3}])(albums: [{4}])</artist>",
+                        artist.UserName, artist.FullName, artist.DateOfBirth, String.Join(", ", artist.genres), String.Join(", ", artist.albumNames));
+                }
+            }
+        }
+
+        public void SaveAlbumListInTextFile()
+        {
+            TextWriter convertAlbumListInTextFile = new StreamWriter(@"\\files\Public\Transfer\Kaloian Karaivanov\Internship\internship_tasks\KokoDajMu\KokoDajMu_MAIN\TextDocuments\AlbumList.txt");
+
+            using (convertAlbumListInTextFile)
+            {
+                foreach (Album album in this.albumsList)
+                {
+                    convertAlbumListInTextFile.WriteLine(" <album><{0}>[{1}](genres: [{2}])(songs: [{3}])<//album>",
+                      album.Name, album.ReleaseDate, String.Join(", ", album.genresList), String.Join(", ", album.songsListNames));
+                }
+            }
+        }
+
+        public void SaveSongListInTextFile()
+        {
+            TextWriter convertSongListInTextFile = new StreamWriter(@"\\files\Public\Transfer\Kaloian Karaivanov\Internship\internship_tasks\KokoDajMu\KokoDajMu_MAIN\TextDocuments\SongList.txt");
+
+            using (convertSongListInTextFile)
+            {
+                foreach (Song song in this.songsList)
+                {
+                    convertSongListInTextFile.WriteLine("<song><{0}>[{1}]</song>", song.Name, song.SongDurationFromTXTFile);
+                }
+            }
+        }
+
+        public void SaveAllFilesInOne()
+        {
+            StreamWriter allFilesWrittenHere = File.CreateText((@"\\files\Public\Transfer\Kaloian Karaivanov\Internship\internship_tasks\KokoDajMu\KokoDajMu_MAIN\TextDocuments\KokoDajMuOverWritten.txt"));
```

Comment:
The path is the same everywhere. Have a constant for it and use it.

Reply:
Also include the txt file in the project and make the path based on the project rather on your machine. should be something like this "../../TextDocuments\SongList.txt"